src/test/java/at/medunigraz/imi/bst/trec/query/DiseaseUmlsSynonymQueryDecoratorTest.java

```diff
@@ -4,22 +4,24 @@
 import at.medunigraz.imi.bst.trec.model.Task;
 import at.medunigraz.imi.bst.trec.model.Topic;
 import at.medunigraz.imi.bst.trec.model.TopicSet;
-import org.junit.Ignore;
+import de.julielab.ir.umls.UmlsSynsetProvider;
 import org.junit.Test;
 
 import java.io.File;
 
-@Ignore
+import static org.assertj.core.api.Assertions.assertThat;
 public class DiseaseUmlsSynonymQueryDecoratorTest {
     @Test
     public void testPmTopics() {
+        UmlsSynsetProvider.setSynsetSourceFile("src/test/resources/umls/melanoma.synset");
+        UmlsSynsetProvider.setUseCache(false);
         DummyElasticSearchQuery dummyQuery = new DummyElasticSearchQuery();
         final DiseaseUmlsSynonymQueryDecorator decorator = new DiseaseUmlsSynonymQueryDecorator(dummyQuery);
         final TopicSet topicSet = new TopicSet(new File("src/main/resources/topics/topics2018.xml"), Challenge.TREC_PM, Task.PUBMED, 2017);
         for (Topic topic : topicSet.getTopics()) {
             decorator.expandTopic(topic);
-            System.out.println(topic.getDisease());
-            System.out.println("   " + topic.diseaseSynonyms);
+            if (topic.getDisease().equals("melanoma"))
+                assertThat(topic.getDiseaseSynonyms()).contains("nevocarcinoma", "cutaneous melanoma", "melanoma (disorder)");
```

Comment:
Lexigram provides here "melanoblastoma", "melanosarcoma", and "melanocarcinoma".

Note that we should filter out "(disorder)" somewhere. There was e.g. some filtering in `Lexigram`.